### DISCUSSION TITLE
How to use template for issues created by clicking "convert to issue"?

### DISCUSSION THREAD

**Discussion by jehtee** [community/community#62935] [New to GitHub]
GH allows you can convert sub-tasks within an issue into their own issues (by clicking the "convert to issue" button).  

This is super helpful.  

But issues created this way don't seem to support using templates.  This is much less helpful.

Is there a way to select a template to use for creating an issue by clicking the "convert to issue" button on a subtask?

**Comment by mehulrajdev** [comment]
Select Create ().
Type a Summary for the issue.
Complete all required fields and any other fields that you want.
Optional: To change which fields appear when you create another issue, select Configure Fields and select and deselect the fields you want. When you next create an issue, the selected fields will be displayed.
Optional: To create other issues with the same Project and Issue Type, select the Create another checkbox. Depending on your configuration and issue creation history, some fields may be pre-populated. Review these before creating a new issue.
When you're finished, select Create.

**Reply by jehtee** [reply]
Thanks.  But this doesn't address my question.

I have several issues templates already.  When I create a new issue from scratch I can select a template.  But when creating an issue from a subtask (by clicking the "Convert to Issue" link) there is no option to select a template.

I want to know if there is a way select a template when converting a subtask to an issue.

**Comment by Julian-Dumitrascu** [comment]
Others, [too](https://github.com/orgs/community/discussions/4287), want this feature.
